bin/powershell/check-windows-disk-writeable.ps1

```diff
@@ -0,0 +1,76 @@
+#
+#   check-windows-disk-writeable.ps1
+#
+# DESCRIPTION:
+#   This plugin collects the mounted disks and tests they are writeable.
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Windows
+#
+# DEPENDENCIES:
+#   PowerShell
```

Comment:
I thought about this but as I just copied `bin/powershell/check-windows-disk.ps1`, thought it would be acceptable.  Will fix this.